u-bmc.go

```diff
@@ -20,97 +20,14 @@ import (
-		"github.com/u-root/u-root/cmds/dhclient",
-		"github.com/u-root/u-root/cmds/dirname",
-		"github.com/u-root/u-root/cmds/dmesg",
-		"github.com/u-root/u-root/cmds/echo",
-		"github.com/u-root/u-root/cmds/elvish",
-		"github.com/u-root/u-root/cmds/false",
-		"github.com/u-root/u-root/cmds/field",
-		"github.com/u-root/u-root/cmds/find",
-		"github.com/u-root/u-root/cmds/free",
-		"github.com/u-root/u-root/cmds/freq",
-		"github.com/u-root/u-root/cmds/gpgv",
-		"github.com/u-root/u-root/cmds/gpt",
-		"github.com/u-root/u-root/cmds/grep",
-		"github.com/u-root/u-root/cmds/gzip",
-		"github.com/u-root/u-root/cmds/hexdump",
-		"github.com/u-root/u-root/cmds/hostname",
-		"github.com/u-root/u-root/cmds/id",
-		"github.com/u-root/u-root/cmds/init",
-		"github.com/u-root/u-root/cmds/insmod",
-		"github.com/u-root/u-root/cmds/installcommand",
-		"github.com/u-root/u-root/cmds/io",
-		"github.com/u-root/u-root/cmds/ip",
-		"github.com/u-root/u-root/cmds/kexec",
-		"github.com/u-root/u-root/cmds/kill",
-		"github.com/u-root/u-root/cmds/lddfiles",
-		"github.com/u-root/u-root/cmds/ln",
-		"github.com/u-root/u-root/cmds/losetup",
-		"github.com/u-root/u-root/cmds/ls",
-		"github.com/u-root/u-root/cmds/lsmod",
-		"github.com/u-root/u-root/cmds/mkdir",
-		"github.com/u-root/u-root/cmds/mkfifo",
-		"github.com/u-root/u-root/cmds/mknod",
-		"github.com/u-root/u-root/cmds/modprobe",
-		"github.com/u-root/u-root/cmds/mount",
-		"github.com/u-root/u-root/cmds/msr",
-		"github.com/u-root/u-root/cmds/mv",
-		"github.com/u-root/u-root/cmds/netcat",
-		"github.com/u-root/u-root/cmds/ntpdate",
-		"github.com/u-root/u-root/cmds/pci",
-		"github.com/u-root/u-root/cmds/ping",
-		"github.com/u-root/u-root/cmds/printenv",
-		"github.com/u-root/u-root/cmds/ps",
-		"github.com/u-root/u-root/cmds/pwd",
-		"github.com/u-root/u-root/cmds/pxeboot",
-		"github.com/u-root/u-root/cmds/readlink",
-		"github.com/u-root/u-root/cmds/rm",
-		"github.com/u-root/u-root/cmds/rmmod",
-		"github.com/u-root/u-root/cmds/rsdp",
-		"github.com/u-root/u-root/cmds/scp",
-		"github.com/u-root/u-root/cmds/seq",
-		"github.com/u-root/u-root/cmds/shutdown",
-		"github.com/u-root/u-root/cmds/sleep",
-		"github.com/u-root/u-root/cmds/sort",
-		"github.com/u-root/u-root/cmds/sshd",
-		"github.com/u-root/u-root/cmds/stty",
-		"github.com/u-root/u-root/cmds/switch_root",
-		"github.com/u-root/u-root/cmds/sync",
-		"github.com/u-root/u-root/cmds/tail",
-		"github.com/u-root/u-root/cmds/tee",
-		"github.com/u-root/u-root/cmds/true",
-		"github.com/u-root/u-root/cmds/truncate",
-		"github.com/u-root/u-root/cmds/umount",
-		"github.com/u-root/u-root/cmds/uname",
-		"github.com/u-root/u-root/cmds/uniq",
-		"github.com/u-root/u-root/cmds/unshare",
-		"github.com/u-root/u-root/cmds/validate",
-		"github.com/u-root/u-root/cmds/vboot",
-		"github.com/u-root/u-root/cmds/wc",
-		"github.com/u-root/u-root/cmds/wget",
-		"github.com/u-root/u-root/cmds/which",
+		"github.com/u-root/u-root/cmds/core/*",
```

Comment:
note that he added some stuff, but let's just see how this works out.

This makes me think the templates should be in the uroot package, not the command, so that we could just use the string
"core"

here. Were that possible the change we made in u-root would not have even been an issue.

Reply:
yeah, I thought the same about templates.